tasks/kernels/build.py

```diff
@@ -0,0 +1,85 @@
+from invoke import task
```

Comment:
This file is ported from https://github.com/faasm/experiment-kernels/blob/master/tasks/build.py